packages/hdwallet-native/src/crypto/isolation/core/bip32/interfaces.ts

```diff
@@ -1,11 +1,12 @@
 import * as core from "@shapeshiftoss/hdwallet-core";
 
-import { Revocable } from "..";
+import { BIP32, Ed25519, Revocable } from "..";
 import * as SecP256K1 from "../secp256k1";
 import { ChainCode } from ".";
 
 export interface Seed extends Partial<Revocable> {
-  toMasterKey(hmacKey?: string | Uint8Array): Promise<Node>;
+  toSecp256k1MasterKey(): Promise<BIP32.Node>;
```

Comment:
Remove the hmac key now that this is defined as secp256k1 (which was always the intention)